tests/always/test_project_structure.py

```diff
@@ -244,10 +243,15 @@ def test_missing_example_for_operator(self):
             }
             all_operators -= example_paths
 
+        covered_but_omitted = self.MISSING_EXAMPLES_FOR_OPERATORS - all_operators
         all_operators -= self.MISSING_EXAMPLES_FOR_OPERATORS
         all_operators -= self.DEPRECATED_OPERATORS
         all_operators -= self.BASE_OPERATORS
-        assert set() == all_operators
+        assert set() == all_operators, (
```

Comment:
Also updated the tests to fail if some operators are set as not covered with examples but example is found.